collections/src/main/scala/strawman/collection/immutable/Vector.scala

```diff
@@ -171,11 +171,6 @@ final class Vector[+A] private[immutable] (private[collection] val startIndex: I
     dropRight(1)
   }
 
-  override def slice(from: Int, until: Int): Vector[A] =
-    take(until).drop(from)
```

Comment:
Removed because `IndexedView.Slice` does the same job without creating an intermediate `Vector` (like `take(until)` does)